eZ/Publish/API/Repository/Values/Content/Search/SearchHit.php

```diff
@@ -43,7 +43,7 @@ class SearchHit extends ValueObject
      *
      * @var string
      */
-    public $contentTranslation;
+    public $matchedTranslation;
```

Comment:
Renamed property in https://github.com/ezsystems/ezpublish-kernel/commit/aa88df6fd49381b7230b754c87916de9b19b189c and https://github.com/ezsystems/ezplatform-solr-search-engine/commit/b50d59e4f3f3e2ae50300a6a09e44868df768115